smithy-openapi/src/main/java/software/amazon/smithy/openapi/OpenApiConfig.java

```diff
@@ -87,6 +84,8 @@ public enum HttpPrefixHeadersStrategy {
             "Homepage", "API Reference", "User Guide", "Developer Guide", "Reference", "Guide");
     private boolean useIntegerType;
 
+    private String version;
```

Comment:
```suggestion
    private String version;
```

You can just set the default version value here

Reply:
modified & resolved.